# Message to represent predicted trajectories of all tracked groups
#
std_msgs/Header header
PredictTraj[] predicted_trajs